tests/gtest/avifgainmaptest.cc

```diff
@@ -47,12 +46,15 @@ void CheckGainMapMetadataMatches(const avifGainMapMetadata& lhs,
 
 avifGainMapMetadata GetTestGainMapMetadata(bool base_rendition_is_hdr) {
   avifGainMapMetadata metadata = {};
-  metadata.backwardDirection = base_rendition_is_hdr;
   metadata.useBaseColorSpace = true;
   metadata.baseHdrHeadroomN = 0;
   metadata.baseHdrHeadroomD = 1;
   metadata.alternateHdrHeadroomN = 6;
   metadata.alternateHdrHeadroomD = 2;
+  if (base_rendition_is_hdr) {
+    std::swap(metadata.baseHdrHeadroomN, metadata.alternateHdrHeadroomN);
+    std::swap(metadata.baseHdrHeadroomD, metadata.alternateHdrHeadroomD);
+  }
```

Comment:
Should we also swap `baseOffset` and `alternateOffset` if `base_rendition_is_hdr` is true?

Reply:
No, this was needed in the JPEG code because they explicitly say "SDR offset" and "HDR offset" in the XMP. When formulated as "base offset" and "alternate offset", no conversion is needed.

Reply:
Thanks for the explanation. Did you mean we store JPEG's "SDR offset" in `metadata.baseOffset` and store JPEG's "HDR offset" in `metadata.alternateOffset`?

I wonder if we should document that because it is not obvious.

Reply:
I was referring to [this code](https://source.chromium.org/chromium/chromium/src/+/main:third_party/libavif/src/apps/shared/avifjpeg.c;l=619;drc=c3809cce7333810f66b79f5319b799422463897e) where we read SDR into "base" and then [swap them](https://source.chromium.org/chromium/chromium/src/+/main:third_party/libavif/src/apps/shared/avifjpeg.c;l=643;drc=c3809cce7333810f66b79f5319b799422463897e) if the base is HDR.